front/src/modules/simulationResult/components/SimulationResultsMap/RenderItinerary.tsx

```diff
@@ -1,6 +1,8 @@
 import type { Feature, LineString } from 'geojson';
-import { Source } from 'react-map-gl/maplibre';
+import { Marker, Source } from 'react-map-gl/maplibre';
 
+import stdcmDestination from 'assets/pictures/stdcm/destination.svg';
+import OriginIcon from 'assets/pictures/stdcm/start.svg';
```

Comment:
Since it is no longer only used in the stdcm map, perhaps you could move the svg to the n-1 folder?

Reply:
Done